samples/application_development/chre/src/main.c

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+
+void main(void)
+{
+	printk("Hello CHRE!\n");
```

Comment:
In which way is this sample using / exemplifying how chre can be used in Zephyr context ?

Reply:
This sample needs to use CHRE in some way, and it should be moved to `samples/modules/chre`.

Also, the sample doesn't build for me. CI didn't catch it because the sample is missing a `sample.yaml`
```
/home/mhelm/zephyrproject/modules/lib/chre/platform/shared/version.cc:29:19: error: expected unqualified-id before string constant
   29 | DLL_EXPORT extern "C" const char _chreVersionString[]
      |                   ^~~
/home/mhelm/zephyrproject/modules/lib/chre/platform/shared/version.cc: In function ‘const char* chre::getChreVersionString()’:
/home/mhelm/zephyrproject/modules/lib/chre/platform/shared/version.cc:34:10: error: ‘_chreVersionString’ was not declared in this scope; did you mean ‘getChreVersionString’?
   34 |   return _chreVersionString;
      |          ^~~~~~~~~~~~~~~~~~
      |          getChreVersionString
```

Reply:
Done